app/jinja_filters.py

```diff
@@ -53,6 +55,10 @@ def format_unit(unit, value=''):
     return units.format_unit(value=value, measurement_unit=unit, length='short', locale=DEFAULT_LOCALE)
 
 
+def as_london_tz(time):
```

Comment:
Let's not call this parameter `time` - it's a datetime. Maybe just call it `value`.